example/readme.md

```diff
@@ -1,81 +0,0 @@
-# Example project
```

Comment:
I think this README is just deleted with no replacement?

Reply:
Was planning on rewriting it but forgot. Added it now (note it acts as if the iOS changes are already applied)